extensions/cornerstone/src/initMeasurementService.ts

```diff
@@ -350,7 +350,13 @@ const connectMeasurementServiceToTools = (
       if (source.name !== CORNERSTONE_3D_TOOLS_SOURCE_NAME) {
         continue;
       }
+      const removedAnnotation = annotation.state.getAnnotation(uid);
       removeAnnotation(uid);
+      commandsManager.run('triggerCreateAnnotationMemo', {
```

Comment:
Is there a way to bulk add them as a single memo?  The undo you undo as a single operation every cleared item all at once.